session/pingpong/accountant_promise_settler.go

```diff
@@ -248,12 +255,57 @@ func (aps *AccountantPromiseSettler) listenForSettlementRequests() {
 		case <-aps.stop:
 			return
 		case p := <-aps.settleQueue:
-			aps.settle(p)
+			aps.settle(p, nil)
 		}
 	}
 }
 
-func (aps *AccountantPromiseSettler) settle(p receivedPromise) {
+// ErrNothingToSettle indicates that there is nothing to settle.
+var ErrNothingToSettle = errors.New("nothing to settle for the given provider")
+
+// ForceSettleAsync forces the settling of promises for the given accountant and provider combo without waiting for the result.
+func (aps *AccountantPromiseSettler) ForceSettleAsync(providerID, accountantID identity.Identity) error {
+	return aps.forceSettle(providerID, accountantID, nil)
```

Comment:
This is not very Go way. If I want to make it async I would like to decide it from calling side.

```go
go ForceSettle(...)
```

It will also simplify internal settle logic.